### ISSUE TITLE
Timings

### ISSUE THREAD

**Issue by andreasnoack** [JuliaParallel/PageRankBenchmark#7]
We should assemble timings such that we can compare them. Later, we might want to do this in a more fancy way but, for now, I've created 

https://docs.google.com/spreadsheets/d/1N82nU3n3cUDdsyIL7lQQ2cjZjr4b9sNtmiu44vMjXTw/edit#gid=0

In case you want to have a look Bradley Kuszmaul's threaded C++ implementation, take a look at his fork of the repo.

**Comment by shashi** [comment]
What is the meaning of `SCALE = [10:22];`? Are we to benchmark for all of those scales?

**Comment by andreasnoack** [comment]
Yes I think so. The layout of the spreadsheet should probably be adjusted to this.
